backend/ee/danswer/background/celery_utils.py

```diff
@@ -38,13 +43,32 @@ def should_perform_chat_ttl_check(
     return True
 
 
-def should_perform_external_permissions_check(
+def should_perform_external_doc_permissions_check(
```

Comment:
would add a comment to say, this does not do the frequency check, the actual sync task checks that and can break out if it doesn't need to sync yet

Reply:
This is only here as a stopgap until richard is done reworking the pruning and i change this to match that (and perhaps move the frequency check elsewhere)